pylib/cqlshlib/cql3handling.py

```diff
@@ -738,11 +738,12 @@ def working_on_keyspace(ctxt):
                     ;
 <whereClause> ::= <relation> ( "AND" <relation> )*
                 ;
-<relation> ::= [rel_lhs]=<cident> ( "[" <term> "]" )? ( "=" | "<" | ">" | "<=" | ">=" | "CONTAINS" ( "KEY" )? ) <term>
+<relation> ::= [rel_lhs]=<cident> ( "[" <term> "]" )? ( "=" | "<" | ">" | "<=" | ">=" | "CONTAINS" ( "KEY" )? | "NOT CONTAINS" ( "KEY" )? ) <term>
```

Comment:
Nit: maybe this can be slightly simplified to:
```suggestion
<relation> ::= [rel_lhs]=<cident> ( "[" <term> "]" )? ( "=" | "<" | ">" | "<=" | ">=" | ( "NOT" )? "CONTAINS" ( "KEY" )?<term>
```

Reply:
I think we are missing `!=`.